services/slam/builtin/builtin_test.go

```diff
@@ -910,12 +910,14 @@ func TestGetMapAndPosition(t *testing.T) {
 
 	createFakeSLAMLibraries()
 
+	mapRate := 200
```

Comment:
make these into a const

Reply:
turns out it cant be a const because constants are not listed as addressable. I've made it a global var instead